src/containers/Layout/MainLayout.tsx

```diff
@@ -53,15 +54,29 @@ const MainLayoutWrapper = styled.div`
 `;
 
 const UstWarning = styled.div`
-  background-color: rgba(233, 61, 68, 1);
+  background-color: rgba(255, 231, 206, 1);
```

Comment:
shouldn't it been added to color palette?

Reply:
Since this is a deprecated component not using our new them I hardcoded them. This whole component will be removed shortly.